evm_loader/lib-interface/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+#![allow(non_camel_case_types)]
+
+pub mod types;
+
+use crate::types::RNeonEVMLibResult;
+use std::{collections::HashMap, path::Path};
+use thiserror::Error;
+
+use abi_stable::{
+    library::{LibraryError, RootModule},
+    package_version_strings,
+    std_types::{RStr, RString},
+    StableAbi,
+};
+
+#[repr(C)]
+#[derive(StableAbi)]
+#[sabi(kind(Prefix(prefix_ref = NeonEVMLib_Ref)))]
+#[sabi(missing_field(panic))]
+pub struct NeonEVMLib {
+    pub hash: extern "C" fn() -> RString,
+    pub get_version: extern "C" fn() -> RString,
```

Comment:
I suggest adding a `LibMethod::BuildInfo` instead of having this `get_version` separate method call.

Reply:
added as additional method